.automation/arch/test_diagram.py

```diff
@@ -0,0 +1,10 @@
+# !/usr/bin/env python3
+"""
+Build architecture diagram of MegaLinter
+"""
+
+from build_arch_diagram import BuildArchDiagram
+
+diag = BuildArchDiagram(29, 11, 13)
+print(diag.get_image_text())
+print(diag.get_image_bin("svg", "docs/assets/images/architecture"))
```

Comment:
This would be a good use case for a snapshot test (e.g., https://github.com/syrusakbary/snapshottest).

Reply:
@nvuillam, I have noticed there are some references to both unittest and pytest in the code base. Do you have a preference for one over the other (or some other testing framework)?